develop-docs/sdk/expected-features/data-handling.mdx

```diff
@@ -63,7 +63,7 @@ This helps Relay to know what kind of data it receives and this helps with scrub
   If query strings are present in the URL or fragments (Browser SDKs only) are present in the URI, both are set into the data attribute of the span.
 
   ```js
-  span.setData({
+  span.setAttributes({
```

Comment:
Only JS exposes `setAttributes`.

Reply:
sad, but JS does not expose `setData` anymore 😅 so if we use a JS example here it should probably be attributes?

Reply:
let’s add both then

Reply:
I added a JS and python section here now?